### DISCUSSION TITLE
If "error", do not submit "value"

### DISCUSSION THREAD

**Discussion by Kornelius777** [jomjol/AI-on-the-edge-device#1240] [Ideas]
Title says it all...
My home automation is having trouble with empty strings.

In case of error - would it be possible to submit the "error" field, but exempt the "value" field?

Thank you kindly!

Kind regards!

**Comment by jomjol** [comment]
Well, this would cause problems in other systems, where an string causes problems.

**Comment by stefanbode** [comment]
I do not see a problem because on current releases the "value" will not change if you checked "use prevalue" and stay on the last good value.

![image](https://user-images.githubusercontent.com/16225290/201065382-125ffae1-5fdf-4223-8e13-69fac604cbff.png)
